mlflow/tracing/trace_manager.py

```diff
@@ -56,13 +57,15 @@ def __init__(self):
             maxsize=MLFLOW_TRACE_BUFFER_MAX_SIZE.get(),
```

Comment:
Can we add a comment explaining what `key` we're expecting here? e.g. request_id -- trace mapping